src/styles/blocks/product-card.css

```
@@ -0,0 +1,95 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 200px;
+  padding-top: 32px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product-card__content {
+  padding: 0 16px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  line-height: 14px;
+  color: var(--secondary);
+  margin-bottom: 16px;
+
+}
+
+.product-card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 24px;
+}
+
+.product-card__review {
+  margin-left: 22px;
+}
+
+.product-card__price-content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 16px;
+}
+
+.product-card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary);
+  margin-right: 85px;
+}
+
+.product-card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main);
+}
+
+.product-card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-basis: 166px;
+  padding: 11px 69px 11px 68px;
```

You could style the element easier, without using flexbox. Just set `display: block` and add vertical paddings.

```suggestion
.product-card__button {
  display: block;
  padding: 11px 0;
```